### PR TITLE
Add container stopped and container down events test

### PR DESCRIPTION
**Pull request by zbud-msft**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?

Add tests for events when container is stopped (sent when container is initially stopped) and when container is down (container has been down for some time)

#### How did you do it?

Add tests

#### How did you verify/test it?

Manual/pipeline

### CODE REVIEW

File: tests/telemetry/events/host_events.py [hunk 1]
```diff
@@ -44,3 +50,39 @@ def trigger_kernel_event(duthost):
...
+        container = "snmp"
+    else:
+        return container
+    container_running = is_container_running(duthost, container)
+    if not container_running:
+        return ""
+    return container
+
+
+def stop_container(duthost):
```
Comment (qiluo-msft) [inline tests/telemetry/events/host_events.py:69]: >stop_container [](http://example.com/codeflow?start=4&length=14)

Do you mean `restart_container`

File: tests/telemetry/events/host_events.py [hunk 2]
```diff
@@ -44,3 +50,39 @@ def trigger_kernel_event(duthost):
     logger.info("Invoking logger for kernel events")
     # syslog at github.com/torvalds/linux/blob/master/fs/squashfs/decompressor_multi.c#L193
     trigger_logger(duthost, "zlib decompression failed, data probably corrupt", "kernel")
+
+
+def get_container(duthost):
```
Comment (qiluo-msft) [inline tests/telemetry/events/host_events.py:55]: >get_container [](http://example.com/codeflow?start=4&length=13)

get_container -> get_running_container

File: tests/telemetry/events/host_events.py (hunk 1 above)
Comment (zbud-msft) [reply tests/telemetry/events/host_events.py]: Changed function name

File: tests/telemetry/events/host_events.py (hunk 2 above)
Comment (zbud-msft) [reply tests/telemetry/events/host_events.py]: Changed function name

**Review by qiluo-msft — APPROVED** [review]

**Comment by wangxin** [comment]
This PR was merged before 202311 branch was created from master branch. No need to cherry-pick for 202311 branch.

**Pull request merged**